assets/js/modules/analytics-4/datastore/report.test.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * `modules/analytics-4` data store: report tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { MODULES_ANALYTICS_4 } from './constants';
+import {
+	createTestRegistry,
+	untilResolved,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import * as fixtures from './__fixtures__';
+
+describe( 'modules/analytics-4 report', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'selectors', () => {
+		describe( 'getReport', () => {
+			const options = {
+				dateRange: 'last-90-days',
```

Comment:
It's worth noting that we don't really use this any more but I believe the API still supports it. We always pass explicit start/end dates now.

Reply:
I switched to `startDate: '2021-01-01' endDate: '2021-01-31', like it's used in the [analytics datastore tests](https://github.com/google/site-kit-wp/blob/63f6d490be64fa3ea6e30592669fb3fcb829e387/assets/js/modules/analytics/datastore/report.test.js#L302-L308)